src/RpcClient.ts

```diff
@@ -354,4 +382,111 @@ export class RpcClient {
       throw new Error(`Error in getSignaturesForAsset: ${error}`);
     }
   }
+  /**
+   * Mint a compressed NFT.
+   * @returns {Promise<MintCNFTResponse>}
+   * @throws {Error}
+   */
+  async mintCNFT(
+    request: MintCNFTRequest
+  ): Promise<MintCNFTResult | undefined> {
+    try {
+      const ownerKeypair = loadWallet(request.ownerKeypair);
+      function delay(ms: number) {
+        return new Promise((resolve) => setTimeout(resolve, ms));
+      }
+      const defaultMetadataArgs: MetadataArgs = {
+        ...request.metadataArgs,
+        primarySaleHappened: false,
+        isMutable: false,
+        editionNonce: 0,
+        tokenStandard: null,
+        uses: null,
+        tokenProgramVersion: null,
+      };
+      let assetId = "";
+      let treeWallet = request.treeKeypair
+        ? loadWallet(request.treeKeypair)
+        : Keypair.generate();
+      if (!request.treeKeypair) {
+        await initTree(this.connection, ownerKeypair, treeWallet);
+      }
+      let response: any | undefined;
+
+      if (!request.collection) {
+        const signature = await mintCompressedNFT(
+          this.connection,
+          defaultMetadataArgs,
+          request.ownerKeypair,
+          treeWallet
+        );
+        await delay(4000);
+        const { id } = await getAssetID(
+          this.connection.rpcEndpoint,
+          treeWallet.publicKey
+        );
+        response = {
+          assetId: id,
+          creator: request.ownerKeypair,
+          treeId: treeWallet.publicKey.toString(),
+          treeKeypair: treeWallet.secretKey,
+          signature: signature,
+        };
+        assetId = id;
+      } else {
+        let collectionMint = request.collectionMint;
+        if (!collectionMint) {
```

Comment:
Wait, so if I say "no collection" a collection will be minted for me? thats not optimal.

If I don't want a collection, it shouldn't mint a collection imo.